securedrop_client/gui/main.py

```diff
@@ -97,6 +100,8 @@ def __init__(self) -> None:
         quit.triggered.connect(self.close)
         self.addAction(quit)
 
+        self.main_view.source_selection_changed.connect(self.source_selection_changed)
```

Comment:
TODO: Add link to Qt docs on connecting signals to signals.

Reply:
Note: @creviera and I also noted that signals and slots would look distinct thanks to the existing (but missing here and there) naming conventions:

- signal: `source_selection_changed`
- corresponding slot: `on_source_selection_changed` (thinking aloud, could probably be `_on_source_selection_changed` since the connection to slots should only happen inside the class... am I correct on that? I can't think of any counter-example.)